src/modules/python.rs

```diff
@@ -46,7 +46,14 @@ pub fn segment(context: &Context) -> Option<Module> {
 
 fn get_python_version() -> Option<String> {
     match Command::new("python").arg("--version").output() {
-        Ok(output) => Some(String::from_utf8(output.stdout).unwrap()),
+        Ok(output) => {
```

Comment:
I think we should try to avoid the `String::from_utf8()` if we don't need it.
`output.stdout` is a `Vec<u8>`, so we should be able to use `output.stdout.is_empty()`.


```rust
if stdout.is_empty() {
	let stdout_string = String::from_utf8(stdout).unwrap();
	return Some(stdout_string);
} else {
	let stderr_string = String::from_utf8(output.stderr).unwrap();
	return Some(stderr_string);
}
```

Reply:
I'd also suggest adding a comment explaining why we are checking both stdout and stderr. 😄

Reply:
@matchai Done!